src/documentation/pages/components/sidenav/examples/_base_lumen_centurylink.vue

```diff
@@ -0,0 +1,512 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base-lumen-centurylink" :tabs="exampleTabs" padding="-p--0">
+    .-d--flex.-overflow--hidden(slot="example")
+      aside.chi-sidenav.-flex--shrink0.-animated(ref="sidenav")
+        .chi-sidenav__content
+          nav
+            ul.chi-sidenav__list
+              li(v-for="val in menuItems" :class="val === activeMenuItem ? '-active' : ''")
+                a(:href="`#drawer-0-${val}`")
+                  i.chi-icon.icon-atom(aria-hidden="true")
+                  span {{ val === activeMenuItem ? 'Active Item' : 'Menu Item' }}
+        .chi-sidenav__drawers
+          .chi-drawer.-animated.-left(v-for="val1 in menuItems" :id="`drawer-0-${val1}`")
+            .chi-drawer__header
+              .chi-drawer__title Menu item {{val1}}
```

Comment:
we could rename val1, val2 variables to more descriptive names like item or activeMenuItem